contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract OETHHarvesterSimple is Governable {
+    ////////////////////////////////////////////////////
+    /// --- STORAGE
+    ////////////////////////////////////////////////////
+    address public operator;
+    address public strategist;
+    mapping(address => bool) public isAuthorized;
+
+    ////////////////////////////////////////////////////
+    /// --- EVENTS
+    ////////////////////////////////////////////////////
+    event Harvested(address token, uint256 amount);
+    event OperatorSet(address operator);
+    event StrategistSet(address strategist);
+    event StrategyStatusSet(address strategy, bool status);
+
+    ////////////////////////////////////////////////////
+    /// --- MODIFIERS
+    ////////////////////////////////////////////////////
+    modifier onlyOperator() {
+        require(
+            msg.sender == operator || isGovernor(),
+            "Only Operator or Governor"
+        );
+        _;
+    }
+
+    ////////////////////////////////////////////////////
+    /// --- CONSTRUCTOR
+    ////////////////////////////////////////////////////
+    constructor(
+        address _governor,
+        address _operator,
+        address _strategist
+    ) {
+        require(_strategist != address(0), "Invalid strategist");
+        operator = _operator;
```

Comment:
These should emit events so we have a full even trace of all strategist and operators.

Reply:
I've removed the `operator` role.